src/quo2/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,296 @@
+
+(defn progress-boxes
+  [networkState]
+  [rn/view
+   {:style style/progress-box-container}
+   (let [numbers (range 1 total-box)]
+     (doall (for [n numbers]
+              [progress-box/progress-bar
+               {:network-state    (calculate-box-state networkState (@app-state :counter) n)
+                :width            "8"
+                :height           "12"
+                :marginHorizontal 2
+                :key              n
+               }])))])
+
+(defn calculate-box-state-arbitrum
+  [networkState]
+  (cond
+    (= networkState "pending") "pending"
+    (= networkState "error")   "error"
+    :else                      "confirmed"))
+
+(defn calculate-box-width
+  [showHalf]
+  (cond
+    (and showHalf (< (@app-state :counter) 30)) (- total-box (@app-state :counter))
+    showHalf                                    30
+    (< (@app-state :counter) total-box)         (- total-box (@app-state :counter))
+    :else                                       0))
+
+(defn progress-boxes-arbitrum
+  [networkState]
+  [rn/view
+   {:style style/progress-box-container}
+   [progress-box/progress-bar
+    {:network-state    (calculate-box-state-arbitrum networkState)
+     :width            "8"
+     :height           "12"
+     :marginHorizontal 2}]
+   [rn/view
+    {:style            style/progress-box-arbitrum
+     :background-color (get-colors "neutral-5")
+     :border-color     (get-colors "neutral-10")}
+    [rn/view
+     (assoc
+      (let [box-style (cond
+                        (= networkState "finalising") (assoc {:style style/progress-box-arbitrum-abs}
+                                                             :right (str (calculate-box-width true) "%")
+                                                             :background-color
+                                                             (colors/custom-color-by-theme :blue 50 60))
+                        (= networkState "finalised")  (assoc {:style style/progress-box-arbitrum-abs}
+                                                             :right (str (calculate-box-width false) "%")
+                                                             :background-color
+                                                             (colors/custom-color-by-theme :blue 50 60))
+                        :else                         (assoc {:style style/progress-box-arbitrum-abs}
+                                                             :background-color
+                                                             (get-colors "neutral-5")))]
+        box-style)
+      :align-self "flex-end"
+      :border-color
+      (get-colors "neutral-10"))]]])
+
+(defn render-text
+  [title override-theme &
+   {:keys [typography weight size style]
+    :or   {typography :main-semibold
+           weight     :semi-bold
+           size       :paragraph-1
+           style      (style/title override-theme)}}]
+  [text/text
```

Comment:
Just a curious question over here.
Are you sure you need to use the markdown component to achieve the design?
I see you're importing `[quo2.components.markdown.text :as text]` and then providing props to modify the text to match the design.
Can you check if the text component already has these variants? that way you won't have to pass these props.
cc @J-Son89

Reply:
@siddarthkay  sorry, I am using as I saw it in many other components. check 1 eg. here `src/quo2/components/wallet/account_card/view.cljs`
I saw `src/quo2/components/markdown/text.cljs` & couldn't find the variants I need,